use-cases/revenue-management.md

```diff
@@ -1,66 +1,101 @@
+
+> **WebSocket disconnection:** In case of WebSocket disconnection, use [Get all reservations](../operations/reservations.md#get-all-reservations-ver-2023-06-06) with `UpdatedUtc` to perform a full resync of reservations that have been updated within the time period since the last full reservation data pull.
+
+The suggested frequency for periodic reservation syncing is 30 minutes for business intelligence solutions. Adjust the frequency depending on your use case. For example, a mobile key integration would rely more on real-time accuracy of reservation data to provide correct door access to the guest, whereas a thirty-minute to one-hour (or longer) interval period might suffice for a business intelligence solution aimed at providing daily statistical insight to a property's business operations.
 
-### Periodic updates and syncing reservation data
+| <div style="width:350px">'How to' use case</div> | API Operations |
+| :-- | :-- |
+| How to listen for changes to reservations | [General Webhooks](../webhooks/wh-general.md) \(`ServiceOrderUpdated` event\) or [WebSockets](../websockets/README.md) \(`Reservation` event\) |
+| How to get reservation details | [Get all reservations](../operations/reservations.md#get-all-reservations-ver-2023-06-06) |
 
-To keep reservations up-to-date and synced across your systems in real time, first cache the reservation data retrieved from the initial reservation data pull, then use [General Webhooks](../webhooks/wh-general.md) \(`ServiceOrderUpdated` event\) or [WebSockets](../websockets/README.md) \(`Reservation` event\) to listen for reservation events.
-When you receive a reservation event, use `ReservationId` obtained from the event to fetch the details of the reservation with [Get all reservations](../operations/reservations.md#get-all-reservations-ver-2023-06-06).
+## Rate pricing
 
-> Note: In case of WebSocket disconnection, use [Get all reservations](../operations/reservations.md#get-all-reservations-ver-2023-06-06) to perform a full resync of reservations that have been updated within the time period since the last full reservation data pull.
+To get rate information, there are two relevant operations. [Get all rates](../operations/rates.md#get-all-rates) gives you information about the names and IDs of the rates in the property, and their status, and also rate groups and restrictions. [Get rate pricing](../operations/rates.md#get-rate-pricing) gives you the pricing of a specific rate for a specific time period. In order to update rate prices, use [Update rate price](../operations/rates.md#update-rate-price). Individual rate, resource category and time span can be selected. To avoid heavy consumption of API resources, while keeping rate price data up-to-date in real time, use [WebSockets](../websockets/README.md) to listen for `PriceUpdate` events. Then retrieve information about the relevant rates using [Get rate pricing](../operations/rates.md#get-rate-pricing), filtering by the relevant `RateId`.
```

Comment:
Following suggestion to support the concurrent change to remove the extent from 'Get all rates' and add new operation 'Get all rate groups' (see https://github.com/MewsSystems/gitbook-connector-api/pull/547):

```suggestion
To get rate information, there are three relevant operations. [Get all rates](../operations/rates.md#get-all-rates) gives you information about the names and IDs of the rates in the property, and their status, and also restrictions. [Get all rate groups](../operations/rategroups.md#get-all-rate-groups) gives you information rate groups. And [Get rate pricing](../operations/rates.md#get-rate-pricing) gives you the pricing of a specific rate for a specific time period. In order to update rate prices, use [Update rate price](../operations/rates.md#update-rate-price). Individual rate, resource category and time span can be selected. To avoid heavy consumption of API resources, while keeping rate price data up-to-date in real time, use [WebSockets](../websockets/README.md) to listen for `PriceUpdate` events. Then retrieve information about the relevant rates using [Get rate pricing](../operations/rates.md#get-rate-pricing), filtering by the relevant `RateId`.
```